hack/local-up-master/openshift-apiserver-manifests/03_apiservice.yaml

```diff
@@ -53,19 +53,6 @@ items:
     insecureSkipTLSVerify: true
     groupPriorityMinimum: 20000
     versionPriority: 15
-- apiVersion: apiregistration.k8s.io/v1
```

Comment:
Any particular reason you're removing it?

Reply:
yeah, it is no longer supported in the latest version of hypershift

Reply:
Good to know, b/c I need to drop that from integration as well.